### PR TITLE
updating Out-* commands with -NoNewLine per Issue 1739

### PR DESCRIPTION
**Pull request by sdwheeler**
updating Out-* commands with -NoNewLine per Issue #1739

Version(s) of document impacted
------------------------------
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

### CODE REVIEW

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Add-Content.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/3.0/Microsoft.PowerShell.Management/Add-Content.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Management/Set-Content.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/3.0/Microsoft.PowerShell.Management/Set-Content.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/3.0/Microsoft.PowerShell.Utility/Out-File.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Utility/Write-Host.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/3.0/Microsoft.PowerShell.Utility/Write-Host.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Management/Add-Content.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/4.0/Microsoft.PowerShell.Management/Add-Content.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Management/Set-Content.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/4.0/Microsoft.PowerShell.Management/Set-Content.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/4.0/Microsoft.PowerShell.Utility/Out-File.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Utility/Write-Host.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/4.0/Microsoft.PowerShell.Utility/Write-Host.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Management/Add-Content.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/5.0/Microsoft.PowerShell.Management/Add-Content.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Management/Set-Content.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/5.0/Microsoft.PowerShell.Management/Set-Content.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/5.0/Microsoft.PowerShell.Utility/Out-File.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Utility/Write-Host.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/5.0/Microsoft.PowerShell.Utility/Write-Host.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Management/Add-Content.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/5.1/Microsoft.PowerShell.Management/Add-Content.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Management/Set-Content.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/5.1/Microsoft.PowerShell.Management/Set-Content.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/5.1/Microsoft.PowerShell.Utility/Out-File.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Utility/Write-Host.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/5.1/Microsoft.PowerShell.Utility/Write-Host.md) | :white_check_mark:Succeeded | |
[reference/6/Microsoft.PowerShell.Management/Add-Content.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/6/Microsoft.PowerShell.Management/Add-Content.md) | :white_check_mark:Succeeded | |
[reference/6/Microsoft.PowerShell.Management/Set-Content.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/6/Microsoft.PowerShell.Management/Set-Content.md) | :white_check_mark:Succeeded | |
[reference/6/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/6/Microsoft.PowerShell.Utility/Out-File.md) | :white_check_mark:Succeeded | |
[reference/6/Microsoft.PowerShell.Utility/Write-Host.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/6/Microsoft.PowerShell.Utility/Write-Host.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C11%5C9%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201711090037438148-1846%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=oZ7dxdONmyhIKg%2BXe5Q19YF%2FzLZ9ht%2BJG%2BBNCNOYSyE%3D&st=2017-11-09T01%3A13%3A06Z&se=2017-12-10T01%3A18%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/3.0/Microsoft.PowerShell.Management/Add-Content.md [hunk 1]
```diff
@@ -9,36 +9,46 @@ title:  Add-Content
...
 The Add-Content cmdlet appends content to a specified item or file.
 You can specify the content by typing the content in the command or by specifying an object that contains the content.
+
 ## EXAMPLES
 
 ### Example 1
-```
+
+```powershell
 PS C:\> add-content -path *.txt -exclude help* -value "END"
```
Comment (zjalexander) [inline reference/3.0/Microsoft.PowerShell.Management/Add-Content.md]: in other versions you cased the examples, but missed these.

File: reference/3.0/Microsoft.PowerShell.Management/Add-Content.md [hunk 2]
```diff
@@ -119,14 +139,17 @@ Accept wildcard characters: False
 ```
 
 ### -Stream
-Adds the content to the specified alternate data stream. If the stream does not yet, exist, Add-Content creates it. Enter the stream name. Wildcards are not supported.
 
-Stream is a dynamic parameter that the FileSystem provider adds to the Add-Content cmdlet. This parameter works only in file system drives.
+Adds the content to the specified alternate data stream. If the stream does not yet, exist, Add-Content creates it.
```
Comment (zjalexander) [inline reference/3.0/Microsoft.PowerShell.Management/Add-Content.md]: missed some semantic linebreaks here

File: reference/3.0/Microsoft.PowerShell.Utility/Out-File.md [hunk 3]
```diff
@@ -168,47 +217,13 @@ Accept pipeline input: True (ByValue)
...
-Default value: 80
-Accept pipeline input: False
-Accept wildcard characters: False
-```
-
 ### -LiteralPath
+
 Specifies the path to the output file.
-Unlike **FilePath**, the value of the **LiteralPath** parameter is used exactly as it is typed.
+Unlike *FilePath*, the value of the *LiteralPath* parameter is used exactly as it is typed.
```
Comment (zjalexander) [inline reference/3.0/Microsoft.PowerShell.Utility/Out-File.md]: In the style guide, we have the following:
>Property names and parameter names should be in bold

File: reference/5.0/Microsoft.PowerShell.Management/Add-Content.md [hunk 4]
```diff
@@ -11,78 +11,88 @@ title:  Add-Content
...
+```powershell
 Add-Content [-Value] <Object[]> [-PassThru] -LiteralPath <String[]> [-Filter <String>] [-Include <String[]>]
  [-Exclude <String[]>] [-Force] [-Credential <PSCredential>] [-WhatIf] [-Confirm] [-UseTransaction]
  [-NoNewline] [-Encoding <FileSystemCmdletProviderEncoding>] [-Stream <String>] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Add-Content** cmdlet appends content to a specified item or file.
+
+The Add-Content cmdlet appends content to a specified item or file.
```
Comment (zjalexander) [inline reference/5.0/Microsoft.PowerShell.Management/Add-Content.md]: Again, according to the style guide:
>When talking about a cmdlet in paragraph, use ` to highlight cmdlet names

**Comment by saldana** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/PowerShell/PowerShell-Docs, working directory: W:\v1ju-s, target branch: staging, target branch commit: b04197f509e8a44a97455bd6d450c1bec14f1772, source branch: 6809f03c7826ebcfb443fe6e60fe46e6/sdw-issue1739, source branch commit: .

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C11%5C13%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201711131953289329-1846%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=S0ZB2bhSoIViE%2B%2BIZ5tPH8R0jdNvqcvF%2FKDudZIgZbI%3D&st=2017-11-13T19%3A54%3A54Z&se=2017-12-14T19%3A59%3A54Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Add-Content.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/3.0/Microsoft.PowerShell.Management/Add-Content.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Management/Set-Content.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/3.0/Microsoft.PowerShell.Management/Set-Content.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Security/Set-Acl.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/3.0/Microsoft.PowerShell.Security/Set-Acl.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Utility/Format-Custom.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/3.0/Microsoft.PowerShell.Utility/Format-Custom.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Utility/Format-Table.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/3.0/Microsoft.PowerShell.Utility/Format-Table.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Utility/Format-Wide.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/3.0/Microsoft.PowerShell.Utility/Format-Wide.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/3.0/Microsoft.PowerShell.Utility/Out-File.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Utility/Write-Host.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/3.0/Microsoft.PowerShell.Utility/Write-Host.md) | :white_check_mark:Succeeded | |
[reference/3.0/Microsoft.PowerShell.Utility/Write-Progress.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/3.0/Microsoft.PowerShell.Utility/Write-Progress.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Management/Add-Content.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/4.0/Microsoft.PowerShell.Management/Add-Content.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Management/Set-Content.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/4.0/Microsoft.PowerShell.Management/Set-Content.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Security/Set-Acl.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/4.0/Microsoft.PowerShell.Security/Set-Acl.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Utility/Format-Custom.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/4.0/Microsoft.PowerShell.Utility/Format-Custom.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Utility/Format-Table.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/4.0/Microsoft.PowerShell.Utility/Format-Table.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Utility/Format-Wide.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/4.0/Microsoft.PowerShell.Utility/Format-Wide.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/4.0/Microsoft.PowerShell.Utility/Out-File.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Utility/Write-Host.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/4.0/Microsoft.PowerShell.Utility/Write-Host.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Utility/Write-Progress.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/4.0/Microsoft.PowerShell.Utility/Write-Progress.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Management/Add-Content.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/5.0/Microsoft.PowerShell.Management/Add-Content.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Management/Set-Content.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/5.0/Microsoft.PowerShell.Management/Set-Content.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Security/Set-Acl.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/5.0/Microsoft.PowerShell.Security/Set-Acl.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Utility/Format-Custom.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/5.0/Microsoft.PowerShell.Utility/Format-Custom.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Utility/Format-Table.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/5.0/Microsoft.PowerShell.Utility/Format-Table.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Utility/Format-Wide.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/5.0/Microsoft.PowerShell.Utility/Format-Wide.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/5.0/Microsoft.PowerShell.Utility/Out-File.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Utility/Write-Host.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/5.0/Microsoft.PowerShell.Utility/Write-Host.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Utility/Write-Progress.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/5.0/Microsoft.PowerShell.Utility/Write-Progress.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Management/Add-Content.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/5.1/Microsoft.PowerShell.Management/Add-Content.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Management/Set-Content.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/5.1/Microsoft.PowerShell.Management/Set-Content.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Security/Set-Acl.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/5.1/Microsoft.PowerShell.Security/Set-Acl.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Utility/Format-Custom.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/5.1/Microsoft.PowerShell.Utility/Format-Custom.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Utility/Format-Table.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/5.1/Microsoft.PowerShell.Utility/Format-Table.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Utility/Format-Wide.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/5.1/Microsoft.PowerShell.Utility/Format-Wide.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/5.1/Microsoft.PowerShell.Utility/Out-File.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Utility/Write-Host.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/5.1/Microsoft.PowerShell.Utility/Write-Host.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Utility/Write-Progress.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/5.1/Microsoft.PowerShell.Utility/Write-Progress.md) | :white_check_mark:Succeeded | |
[reference/6/Microsoft.PowerShell.Management/Add-Content.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/6/Microsoft.PowerShell.Management/Add-Content.md) | :white_check_mark:Succeeded | |
[reference/6/Microsoft.PowerShell.Management/Set-Content.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/6/Microsoft.PowerShell.Management/Set-Content.md) | :white_check_mark:Succeeded | |
[reference/6/Microsoft.PowerShell.Security/Set-Acl.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/6/Microsoft.PowerShell.Security/Set-Acl.md) | :white_check_mark:Succeeded | |
[reference/6/Microsoft.PowerShell.Utility/Format-Custom.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/6/Microsoft.PowerShell.Utility/Format-Custom.md) | :white_check_mark:Succeeded | |
[reference/6/Microsoft.PowerShell.Utility/Format-Table.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/6/Microsoft.PowerShell.Utility/Format-Table.md) | :white_check_mark:Succeeded | |
[reference/6/Microsoft.PowerShell.Utility/Format-Wide.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/6/Microsoft.PowerShell.Utility/Format-Wide.md) | :white_check_mark:Succeeded | |
[reference/6/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/6/Microsoft.PowerShell.Utility/Out-File.md) | :white_check_mark:Succeeded | |
[reference/6/Microsoft.PowerShell.Utility/Write-Host.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/6/Microsoft.PowerShell.Utility/Write-Host.md) | :white_check_mark:Succeeded | |
[reference/6/Microsoft.PowerShell.Utility/Write-Progress.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1739/reference/6/Microsoft.PowerShell.Utility/Write-Progress.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C11%5C13%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201711132030065751-1846%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=QkDqdzfIrHF1aI2l2MfEnxG%2Fi9VMNG3swlaPEDhbdu4%3D&st=2017-11-13T20%3A56%3A13Z&se=2017-12-14T21%3A01%3A13Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**